cockatrice/src/userinfobox.cpp

```diff
@@ -64,16 +64,18 @@ UserInfoBox::UserInfoBox(AbstractClient *_client, bool _editable, QWidget *paren
         connect(&avatarButton, SIGNAL(clicked()), this, SLOT(actAvatar()));
     }
 
-    setWindowTitle(tr("User information"));
+    setWindowTitle(tr("User Information"));
     setLayout(mainLayout);
     retranslateUi();
+    setFixedHeight(sizeHint().height());
+    setFixedWidth(sizeHint().width());
```

Comment:
do we want this?

Reply:
It's more like a personal preference. It would need some tweaking anyway as the later updated user level could be cut off.
Removed it again.